bash_scripts/run_sim.sh

```diff
@@ -1,40 +1,68 @@
 #!/bin/bash
 
-#SBATCH -p cpu-long
-#SBATCH -c 1
+#SBATCH -p arm-preempt
+#SBATCH -c 20
 #SBATCH -G 0
-#SBATCH --mem=16000
-#SBATCH -t 1-00:00:00
-#SBATCH -o slurm-%j.out
+#SBATCH --mem=40000
+#SBATCH -t 14-00:00:00
+#SBATCH -o slurm-%A_%a.out  # Output file for each task
+#SBATCH --array=0-3  # Define array size based on total combinations
 
-# This script is designed to run a non-artificial intelligence simulation on the Unity cluster at UMass Amherst.
-# Users can provide custom parameters via the command line using -- before the parameter list.
+set -e  # Stop the script if any command fails
 
-# Ensure the script stops if any command fails
-set -e
+echo "Starting job with SLURM_ARRAY_TASK_ID: $SLURM_ARRAY_TASK_ID"
 
-# Change to the home directory and then to the SDN simulator directory
-# shellcheck disable=SC2164
-cd
-# shellcheck disable=SC2164
-cd /work/pi_vinod_vokkarane_uml_edu/git/sdn_simulator/
+# Load the Python 3.11.7 module
+module load python/3.11.7
 
-# Load the required Python module
-module load python/3.11.0
+# Verify the loaded Python version
+python --version
 
-# Activate the virtual environment
-source venvs/unity_venv/venv/bin/activate
```

Comment:
I think the venv is only used for RL, just leaving a note so I can remember to double check.